porcupine/plugins/filetypes.py

```diff
@@ -252,6 +253,20 @@ def parse_filetype_name(name: str) -> FileType:
     )
 
 
+filetypes_var: tkinter.StringVar | None = None
+
+
+def _sync_filetypes_menu(event: object) -> None:
+    tab = get_tab_manager().select()
+    if isinstance(tab, tabs.FileTab) and tab.path is not None:
+        try:
+            filetype_name = tab.settings.get('filetype_name', str)
+            if filetypes_var and filetype_name:
+                filetypes_var.set(filetype_name)
+        except KeyError:
+            pass
```

Comment:
Instead of doing nothing, we should set some dummy value (e.g. empty string) to the filetypes var, so that the menu will display no item as being selected.